adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -1034,19 +1063,18 @@ def _wait_for_msg(self, timeout=0.1):
 
         return res[0]
 
-    def _recv_len(self):
+    def _recv_len(self) -> int:
         """Unpack MQTT message length."""
         n = 0
         sh = 0
-        b = bytearray(1)
         while True:
             b = self._sock_exact_recv(1)[0]
             n |= (b & 0x7F) << sh
             if not b & 0x80:
                 return n
             sh += 7
 
-    def _sock_exact_recv(self, bufsize):
+    def _sock_exact_recv(self, bufsize: int) -> bytearray:
         """Reads _exact_ number of bytes from the connected socket. Will only return
         string with the exact number of bytes requested.
```

Comment:
Can we update this docstring be to:

```
"Will only return bytearray with the exact number..."
```
It references string currently but looks to return a bytearray from the code and typing annotation added.